mixer/test/client/env/envoy.go

```diff
@@ -44,89 +42,73 @@ func (s *TestSetup) NewEnvoy() (*Envoy, error) {
 
 	debugLevel := ev.RegisterStringVar("ENVOY_DEBUG", "info", "Specifies the debug level for Envoy.").Get()
 
-	baseID := ""
-	args := []string{"-c", confPath,
-		"--drain-time-s", "1",
-		"--allow-unknown-fields"}
```

Comment:
nit: flag has been deprecated for awhile

Reply:
good catch ... sent out https://github.com/istio/istio/pull/17191 to fix.